### PR TITLE
Support ContinueHandling

### PR DESCRIPTION
**Pull request by byehack**
basic implement for https://github.com/eternnoir/pyTelegramBotAPI/pull/1704

- [ ] Test
- [ ] Create example
- [ ] Document
- [ ] Bump new version

### CODE REVIEW

**Comment by Badiboy** [comment]
image

**Comment by Badiboy** [comment]
image

It is not the subject of your update.

**Comment by Badiboy** [comment]
Looks fine now. 
@coder2020official ?

File: telebot/async_telebot.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # storages
 from telebot.asyncio_storage import StateMemoryStorage, StatePickleStorage, StateStorageBase
-from telebot.asyncio_handler_backends import BaseMiddleware, CancelUpdate, SkipHandler, State
+from telebot.asyncio_handler_backends import BaseMiddleware, CancelUpdate, SkipHandler, State, ContinueHandling
```
Comment (coder2020official) [inline telebot/async_telebot.py:17]: You forgot to create a duplicate of ContinueHandling for async

**Review by coder2020official — CHANGES_REQUESTED** [review]

**Comment by Badiboy** [comment]
@coder2020official 
Looks fine now.

File: telebot/handler_backends.py [hunk 2]
```diff
@@ -265,6 +261,15 @@ class CancelUpdate:
...
     """
+    def __init__(self) -> None:
+        pass
+
 
+class ContinueHandling:
+    """
+    Class for continue updates in handlers.
+    Just return instance of this class 
+    in handlers to continue process.
```
Comment (coder2020official) [inline telebot/handler_backends.py:272]: Add code block here, something like this:

```suggestion
    in handlers to continue process.
    
    .. code-block:: python3
        :caption: Example of using ContinueHandling

        @bot.message_handler(commands=['start'])
        def start(message):
            bot.send_message(message.chat.id, 'Hello World!')
            return ContinueHandling()
        
        @bot.message_handler(commands=['start'])
        def start2(message):
            bot.send_message(message.chat.id, 'Hello World2!')
    
```

Same with async one but with async syntax.

File: telebot/__init__.py [hunk 3]
```diff
@@ -37,7 +37,10 @@
 logger.setLevel(logging.ERROR)
 
 from telebot import apihelper, util, types
-from telebot.handler_backends import HandlerBackend, MemoryHandlerBackend, FileHandlerBackend, BaseMiddleware, CancelUpdate, SkipHandler, State
+from telebot.handler_backends import (
```
Comment (coder2020official) [inline telebot/__init__.py:40]: What about examples of this feature on async and sync for examples folder?

**Review by coder2020official — CHANGES_REQUESTED** [review]

**Comment by byehack** [comment]
@Badiboy @coder2020official 
You have access to edit this PR.
As I don't know how to do tasks below, I recommend you doing each task below then check mark them:


> * [ ]  Test
> * [ ]  Create example
> * [ ]  Document
> * [ ]  Bump new version

**Comment by Badiboy** [comment]
@coder2020official I propose to finish this PR "as is". It is functionally done and waits too much. Other tests and documentation can be done via new PR...

**Comment by coder2020official** [comment]
Ig that's it. Checked documentation - everything is fine.

**Review by coder2020official — APPROVED** [review]

**Comment by Badiboy** [comment]
Come on, guys, let's go )

**Comment by coder2020official** [comment]
Ура, бабахнули в мастер

Thanks for patience @byehack

**Comment by coder2020official** [comment]
* Just a reminder that we have English chat:
* https://github.com/byehack/pyTelegramBotAPI/tree/ContinueHandling#the-telegram-chat-group

**Pull request merged**